sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/java/com/azure/ai/inkrecognizer/InkStroke.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.azure.ai.inkrecognizer;
+
+/**
+ * The InkStroke interface represents an ink stroke (a collection of ink points from the time a user places his writing instrument on
```

Comment:
I'd say "the" instead of his to be gender neutral

Reply:
Fixed.